rcon/commands.py

```diff
@@ -77,11 +79,17 @@ class ServerCtl:
     """
 
     def __init__(self, config, auto_retry=1):
+        # .env fed config from rcon.SERVER_INFO
         self.config = config
         self.conn = None
         # self._connect()
         self.auto_retry = auto_retry
 
+        # config/default_config.yml config.yml, etc.
+        config = get_config()
+        # logger.debug(f"{config=}")
+        self.advanced_settings = AdvancedConfigOptions(**config['ADVANCED_CRCON_SETTINGS'])
```

Comment:
Since this layer doesn't use it I'd have put that in the init of the extended one. Now this class can't be used with a config it's a bit of a shame.